src/components/screens/profile/ProfileDetailsForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Button, Spacer, Row } from '@baca/design-system'
+import { useUpdateProfileForm } from '@baca/hooks'
+import { useRouter } from 'expo-router'
+import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { ProfileControlledInput } from './ProfileControlledInput'
+
+export const ProfileDetailsForm = () => {
+  const { t } = useTranslation()
+  const { control, errors, setFocus, submit, isSubmitting } = useUpdateProfileForm()
+  const { back } = useRouter()
+
+  const focusLastNameInput = useCallback(() => setFocus('lastName'), [setFocus])
+
+  return (
+    <Box>
+      <Box borderColor="border.secondary" borderTopWidth={1} py={6}>
+        <ProfileControlledInput
+          label={t('form.labels.first_name')}
+          name="firstName"
+          placeholder={t('form.placeholders.first_name')}
+          control={control}
```

Comment:
could be `{...{control, errors}}`  here ;)

Reply:
Nice rest operator here again, thanks Michał 💻